extensions/usage-metrics-dashboard/Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: test update-manifest
+
+test:
+	Rscript -e 'testthat::test_package(".")'
```

Comment:
I cannot seem to get this running. I installed R `4.3` to match the lockfile, setup an renv, and installed `testthat` directly.

When running `testthat::test_package(".")` or `make test` I get:

```
No installed testthat tests found for .
```

Reply:
I'd recommend including a `CONTRIBUTING.md` file with instructions.

Reply:
Thanks for catching this. The workflow changed just earlier today but I forgot to update the makefile.